Sources/HummingbirdWSCore/WebSocketHandler.swift

```diff
@@ -166,13 +167,15 @@ package actor WebSocketHandler {
                 handler: self
             )
             let closeCode: WebSocketErrorCode
+            var clientError: Error?
             do {
                 // handle websocket data and text
                 try await handler(webSocketInbound, webSocketOutbound, context)
                 closeCode = .normalClosure
             } catch InternalError.close(let code) {
                 closeCode = code
             } catch {
```

Comment:
```suggestion
            } catch where type == .server {
```

Can't we do something like this instead?

Reply:
It doesn't really help, in that we have to repeat the close handshake in both cases.